### ISSUE TITLE
Not necessarily an issue, but would be cool if possible.

### ISSUE THREAD

**Issue by Koraxor** [jesuscc1993/gbf-bookmarker#7]
If it's possible to use your extension with this other Granblue extension (Granblue POPUP https://chromewebstore.google.com/detail/granblue-popup/amfpomjjdhemljbglnejdlmoaolakpib?hl=en) that would be pretty cool, the shortcut keybinds do work in the POPUP window but I am unable to open the UI of the GBF Bookmarker in the POPUP window, even if using the activate extension keybind.

I don't know if this would be possible from your end, or if you have an idea of how I could do it. Or possibly some way to extract your bookmarks? (POPUP does have a sort of bookmark system you can access by double right clicking in game but you have to populate it yourself, and it's very basic. Also the JST time on yours is pretty useful lol.

I'm sorry, I know it's probably not on you, but I've messaged the creator before (while not about this, but about some other possible features) and never got a response. If you don't have the time or don't even want to that's cool, I still get use out of the keybinds, but I think it would be really nice to use the two together.

**Comment by jesuscc1993** [comment]
The problem is popup windows don't load the extension icons and they can't load their popup menus.
I tried opening the extension on a detached popup, but then the problem is it doesn't share a context with the other chrome window and it can't change it's address.
TL;DR don't think I can load a working interface into a popup window.

**Comment by Koraxor** [comment]
No worries then, thank you for trying!
